### PR TITLE
Corrects build process for node-webkit app under OS X

### PR DESCRIPTION
**Pull request by justinjams**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,30 +58,27 @@ $ cd source
...
 > copy /b nw.exe+moonshot.nw moonshot.exe
+```
 
-OS X:
+Or, for OS X, copy the files to node-webkit.app/Contents/Resources/app.nw
+
+```
 $ cd source
-$ zip -r ../moonshot.nw *
+$ rsync -r -R * ../node-webkit.app/Contents/Resources/app.nw
```
Comment (mildmojo) [inline README.md]: Why favor `rsync` over `cp`? Asking for a friend.

Comment (mildmojo) [inline README.md]: (e.g. `cp -R ./ ../node-webkit.app/Contents/Resources/app.nw/`)

Comment (justinjams) [inline README.md]: Simply, it came to me first. I couldn't give a better explanation. *nix CLI utilities are not my strong suit. Would you prefer using `cp`?

Comment (mildmojo) [inline README.md]: For that operation, yeah. Looks like `rsync` is actually part of OS X, but `cp` is more idiomatic.

**Comment by mildmojo** [comment]
Is this PR still relevant, or was it included in #7?

**Comment by justinjams** [comment]
Nope. That's why I closed this one... right about now

**Pull request closed without merging**